ubuntu-classic-2504-amd64.json

```diff
@@ -6,7 +6,7 @@
     "model": "ubuntu-classic-2504-amd64",
     "architecture": "amd64",
     "timestamp": "2024-10-30T12:00:00.0Z",
-    "base": "core22",
+    "base": "core24",
```

Comment:
Does this work without adding core24 to the model explicitly?

Reply:
Apparently so?  Seems safer to list it explicitly.